docs/Maintainers.rst

```diff
@@ -68,6 +68,10 @@ The EasyBuild maintainers all have admin access to the different EasyBuild GitHu
 
   * GitHub: `@migueldiascosta <https://github.com/migueldiascosta>`_ - e-mail: ``migueldiascosta (at) nus.edu.sg`` - IRC/Slack: ``migueldiascosta``
 
+* **Alex Domingo** (`Free University of Brussels <https://hpc.vub.be>`_)
```

Comment:
```suggestion
* **Alex Domingo** (`Vrije Universiteit Brussel <https://hpc.vub.be>`_)
```

Reply:
We are told to not use the english translation to avoid confusion with ULB

Reply:
OK, then please fix for @wpoely86 too...

Reply:
Actually, I'll do that myself...